### PR TITLE
added gtid_executed vitess show query

### PR DESCRIPTION
**Pull request by harshit-gangal**
Signed-off-by: Harshit Gangal <harshit@planetscale.com>

### CODE REVIEW

**Review by shlomi-noach — APPROVED** [review]

**Comment by shlomi-noach** [comment]
If this is not part of v10, then please don't merge until we've released v10

**Comment by harshit-gangal** [comment]
ACK

File: content/en/docs/reference/compatibility/mysql-compatibility.md [hunk 1]
```diff
@@ -173,3 +236,6 @@ type DBDDLPlugin interface {
 It must then register itself calling `DBDDLRegister`. 
 You can take a look at the `dbddl_plugin.go` in the engine package for an example of how it's done.
 Finally, you need to add a command line flag to vtgate to have it use the new plugin: `-dbddl_plugin=myPluginName`
+
```
Comment (systay) [inline content/en/docs/reference/compatibility/mysql-compatibility.md:239]: were these new lines intentionally added?

**Review by systay — APPROVED** [review]

File: content/en/docs/reference/compatibility/mysql-compatibility.md (hunk 1 above)
Comment (harshit-gangal) [reply content/en/docs/reference/compatibility/mysql-compatibility.md:239]: yes

**Pull request merged**